internal/cmd/commands/authenticate/authenticate.go

```diff
@@ -35,5 +48,42 @@ func (c *Command) Help() string {
 }
 
 func (c *Command) Run(args []string) int {
-	return cli.RunResultHelp
+	client, err := c.Client(base.WithNoTokenScope(), base.WithNoTokenValue())
+	if c.WrapperCleanupFunc != nil {
+		defer func() {
+			if err := c.WrapperCleanupFunc(); err != nil {
+				c.PrintCliError(fmt.Errorf("Error cleaning kms wrapper: %w", err))
+			}
+		}()
+	}
+	if err != nil {
+		c.PrintCliError(fmt.Errorf("Error creating API client: %w", err))
+		return base.CommandCliError
+	}
+
+	// Lookup the primary auth method ID in the global scope
+	aClient := authmethods.NewClient(client)
+	pri, err := getPrimaryAuthMethodId(c.Context, aClient, "global", "")
+	if err != nil {
+		c.PrintCliError(errors.New("Error looking up primary auth method ID for global scope. Try setting primary auth method for global scope, or use a auth method subcommand (see 'boundary authenticate -h' for available sub command usage)."))
```

Comment:
Should we not be using the definite article for the global scope?
```suggestion
		c.PrintCliError(errors.New("Error looking up primary auth method ID for the global scope. Try setting a primary auth method for the global scope, or use an auth method subcommand (see 'boundary authenticate -h' for available sub command usage)."))
```